internal/services/datadog/azurerm_datadog_monitor_sso_configuration.go

```diff
@@ -0,0 +1,172 @@
+package datadog
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/datadog/mgmt/2021-03-01/datadog"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/datadog/parse"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/datadog/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/timeouts"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+func resourceDatadogSingleSignOnConfigurations() *pluginsdk.Resource {
```

Comment:
since there appears to be a 1:1 relationship with a DataDog Monitor - I think this likely should be embedded within that resource, rather than a separate resource in it's own right

Reply:
We have dependency to create the SSO, depending on the way monitor is created.
So including this in monitors block won't be feasible.

Reply:
@tombuildsstuff please let me know if more information is needed.

Reply:
> We have dependency to create the SSO, depending on the way monitor is created. So including this in monitors block won't be feasible.

could you explain this? what do you mean create the SSO?

Reply:
During the linking flow, SigleSignOn can not be configured, but can be updated later.
In new org creation workflow, SingleSignOn in optional in create flow. This also can be updated later by user.

Reply:
@katbyte does this answer the question?